nautobot/virtualization/tables.py

```diff
@@ -223,6 +225,8 @@ class Meta(BaseTable.Meta):
             "pk",
             "name",
             "enabled",
+            "parent_interface",
+            "bridge",
             "mac_address",
             "mtu",
             "mode",
```

Comment:
In DeviceInterfaceTable we added `parent_interface` to `default_columns`, but you haven't done so here - intentional?